src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -718,87 +717,80 @@ private void testDatePickerScripts() {
         feedbackPage.clickCustomPublishTimeButton();
 
         // setup various dates
-        Calendar initialCal = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
-        SimpleDateFormat sdf = new SimpleDateFormat("EEE MMM dd yyyy");
-        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
-        initialCal.set(2014, 3, 16, 0, 0, 0);
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEE MMM dd yyyy");
```

Comment:
~This is a rather strange datetime pattern. `Thu, Apr 12 2018`? The datepicker on the UI produces dates with the format `EEE, dd MMM, yyyy`. It works, but should the getter functions below like `getValueOfDate`, `getMaxDateOf` and `getMinDateOf` be refactored to match what the UI actually produces?~

Since this tests `datepicker.js`...using java code..., we may not want to do the refactoring and delegate it to the whole js-unit-test issue #8749